pulsar-functions/runtime-all/pom.xml

```diff
@@ -86,6 +86,15 @@
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-core</artifactId>
     </dependency>
+
+    <!-- avro -->
+    <dependency>
+      <groupId>org.apache.avro</groupId>
```

Comment:
see https://github.com/apache/pulsar/issues/11274
for this change.

without adding Avro to the Pulsar IO classpath it is not possible to access the Native Object